gnomenews/tracker.py

```diff
@@ -44,16 +44,32 @@ def __init__(self):
         self.sparql = Trackr.SparqlConnection.get(None)
 
     @log
+    def _build_get_post_query(self):
+        """Build the standard SELECT msg query
+        """
+
+        return """
+            SELECT
+              nie:url(?msg) AS url
+              nie:title(?msg) AS title
+              nco:fullname(?creator) AS fullname
+              nie:url(?website) AS author_homepage
+              nco:emailAddress(?email) AS author_email
+              nie:contentCreated(?msg) AS date
+              nmo:htmlMessageContent(?msg) AS content
+        """
+
+    @log
+    def _build_optional(self):
+        return """
+            OPTIONAL { ?msg nco:creator ?creator .
+            OPTIONAL { ?creator nco:hasEmailAddress ?email } .
```

Comment:
a bit more indent please for this and next string (it's part of previous OPTIONAL